.projenrc.ts

```diff
@@ -679,7 +679,9 @@ const tmpToolkitHelpers = configureProject(
     ],
     deps: [
       cloudAssemblySchema.name,
+      cloudFormationDiff,
```

Comment:
elsewhere we are depending on `cloudformationDiff.customizeReference({ versionType: 'exact' }),` and i'm not sure why. just thought i'd flag